modules/ossm-installing-multi-primary-network-mesh.adoc

```diff
@@ -3,9 +3,9 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="ossm-installing-multi-primary-multi-network-mesh_{context}"]
-= Installing a multi-primary multi-network mesh 
+= Installing the multi-primary multi-network mesh 
```

Comment:
I think this should be "a", not "the", since you're installing one mesh of this type. If the sentence read "multi-primary multi-network mesh topology", then it would be "the", right?

Reply:
done